flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -1018,6 +1023,11 @@ public final boolean isRunning() {
         return isRunning;
     }
 
+    @VisibleForTesting
+    public final boolean isRestoring() {
```

Comment:
I actually don't see that this is used in tests. I think you can remove this method